src/components/navBar.tsx

```diff
@@ -27,25 +32,34 @@ export default function NavBar() {
   }, [clickedButton]);
 
   return (
-    <div className="NavBar mx-auto flex h-fit w-[98%] justify-between px-16 py-6">
+    <div className="NavBar mx-auto flex h-fit w-[98%] justify-between px-8 py-6 sm:px-16">
       <Link href={"/"}>
         <Image
           className="Logo mt-4 max-w-44 cursor-pointer transition-transform duration-300 hover:scale-105"
           src="/sfjs-logo.svg"
           width={229}
           height={75}
           alt="Straightforward Job Site logo"
+          onClick={() => setClickedButton("main")}
         />
       </Link>
 
-      <div className="NavButtonContainer flex items-center gap-6">
+      <div className="NavButtonContainer hidden items-end gap-4 sm:flex sm:flex-row sm:items-center sm:max-md:-mr-8 md:gap-6">
         <SiteButton
           variant="filled"
           colorScheme="b4"
           aria="sign up"
           value="signup"
-          onClick={handleNavButtonClick}
+          onClick={(e) => {
```

Comment:
writing this with a ternary operator would make this cleaner. Otherwise I think it would be better to define the function above
```suggestion
          onClick={(e) => {window.innerWidth < 1024 ? showModal(<SignupOptionsModal />) : handleNavButtonClick(e)}}